app/src/main/java/de/westnordost/streetcomplete/data/meta/OsmTaggings.kt

```diff
@@ -35,7 +35,7 @@ const val SURVEY_MARK_KEY = "check_date"
 // generated by "make update" from https://github.com/mnalis/StreetComplete-taginfo-categorize/
 val KEYS_THAT_SHOULD_BE_REMOVED_WHEN_SHOP_IS_REPLACED = listOf(
     "shop_?[1-9]?(:.*)?", "craft_?[1-9]?", "amenity_?[1-9]?", "old_amenity", "old_shop",
-    "information", "leisure", "office", "tourism",
+    "information", "leisure", "office", "tourism", "craft", "shop", "amenity",
```

Comment:
Actually I think this is a bug on current master, no? `amenity=cafe`  should certainly be removed when a café is replaced with a florist or the other way round??

Reply:
@westnordost Aren't those matched by regexps above? ie. `craft_?[1-9]?` should match  `craft` as well as `craft_1` and `craft2` (same for `shop` /  `amenity`). At least if kotlin regexps work like I'm used to in `grep(1)` etc. ?

Reply:
Oh, right. It seems I overlooked the `? `